libmachine/mcnutils/b2d.go

```diff
@@ -307,14 +307,16 @@ func (b *b2dISO) version() (string, error) {
 
 	fullVersion := string(isoMetadata)
```

Comment:
Do you reckon this could just be changed to `strings.TrimSpace(string(isoMetadata))`?